### ISSUE TITLE
Disappearing URL

### ISSUE THREAD

**Issue by JonSilver** [subasraj/flashpost-support#35]
The URL has disappeared for me on all my queries, including new ones, rendering the entire plugin useless for everything except executing established saved queries.

Version: v1.1.79

![image](https://github.com/subasraj/flashpost-support/assets/8831106/76763bd6-500d-4883-af81-7ebde09d3566)

## Runtime Status

![image](https://github.com/subasraj/flashpost-support/assets/8831106/70185539-b94e-4bb9-821a-7086b5167a38)

**Comment by subasraj** [comment]
Is it possible to send me the collection to debug?  Something got deleted or removed in the request.

**Comment by JonSilver** [comment]
I'd rather not upload it to here due to sensitivity of contents, but could send it to you in some other more direct way.

**Comment by subasraj** [comment]
Not an issue. I was able to open the requests.

**Comment by JonSilver** [comment]
It is an issue, because I can open all the requests, but not see or set the URL.
Error messages are error messages because something is going wrong in the code that's not being caught. I showed you the error messages. They're from your code, not mine, not Microsoft's. Hence I reported it here.
I uninstalled and reinstalled the extension. Nothing changed.
So we now can't use your extension any more.
Back to PostMan, I guess.
